classes/gui/ScatterView.sc

```diff
@@ -265,69 +336,92 @@ ScatterView3d {
 		this.data = data;
 		this.refresh
 	}
+
 	data_{|data|
 		data3d = data.collect {|item|
 			Matrix.withFlatArray(3, 1, [
-				specX.unmap(item[0]), 
-				specY.unmap(item[1]), 
+				specX.unmap(item[0]),
+				specY.unmap(item[1]),
 				specZ.unmap(item[2])
 			] * 2 - 1);
 		};
 		scatterView.data = this.pr_project;
 
 //		highlightItem = min(highlightItem, (adjustedData.size-1));
 	}
+
 	refresh {
 		scatterView.refresh;
 	}
+
 	rotX_{|val|
 		this.rot(val, rotY, rotZ);
 	}
+
 	rotY_{|val|
 		this.rot(rotX, val, rotZ);
 	}
+
 	rotZ_{|val|
 		this.rot(rotX, rotY, val);
 	}
+
+	/*
```

Comment:
why are there these pseudo doxygen inline documentations? is this new in 3.10.?

Reply:
haha, no.. this is my own javadoc-like syntax that I'm starting to use. I'm slowly making a similar tool written in SC that can convert these kinds of comments to the helpfile format